electrumx/lib/util.py

```diff
@@ -41,9 +41,14 @@
     import rapidjson as json
 except ImportError:
     try:
-        import ujson as json
+        import orjson as json
     except ImportError:
-        import json
+        try:
+            import ujson as json
+        except ImportError:
+            import json
+
+print("using", json.__name__)
```

Comment:
Can you remove this print statement for now? It should eventually be a `logger.debug` from one of the modules responsible for environment on startup.